The SDK_CMakeLists.txt.in file does not exist at the moment (sed - 2018-04-25)

--- a/Samples/CMakeLists.txt	2018-04-25 07:30:38.211048775 +0200
+++ b/Samples/CMakeLists.txt	2018-04-25 07:32:32.842049802 +0200
@@ -97,10 +97,6 @@
 	PATTERN "scripts" EXCLUDE
   )
   # install a new CMakeLists.txt file to allow building of samples
-  configure_file(${OGRE_TEMPLATES_DIR}/SDK_CMakeLists.txt.in ${CMAKE_CURRENT_BINARY_DIR}/../CMakeLists.txt @ONLY)
-  configure_file(${OGRE_TEMPLATES_DIR}/SDK_Samples_CMakeLists.txt.in ${CMAKE_CURRENT_BINARY_DIR}/CMakeLists.txt @ONLY)
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/../CMakeLists.txt DESTINATION ${OGRE_SAMPLES_DIR}/../)
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/CMakeLists.txt DESTINATION ${OGRE_SAMPLES_DIR})
 endif ()
 
 # Install sample headers, some people rely on these
